ArduCopter/mode.h

```diff
@@ -779,7 +779,7 @@ class ModeAuto : public Mode {
         float down;   // desired speed downwards in m/s. 0 if unset
     } desired_speed_override;
 
-    float circle_last_num_complete;
+    int circle_last_num_complete;
```

Comment:
I did consider doing in this way.

I think there's scope for printing partial circles if you are circling slowly, which is why I went with float here.

Reply:
The number of rotations must be an integer. For example, 0.5 rotations cannot be specified.